ee/clickhouse/queries/funnels/funnel_event_query.py

```diff
@@ -77,4 +77,4 @@ def _get_entity_query(self, entities=None, entity_name="events") -> Tuple[str, D
             else:
                 events.add(entity.id)
 
-        return f"AND event IN %({entity_name})s", {entity_name: list(events)}
+        return f"AND event IN %({entity_name})s", {entity_name: sorted(list(events))}
```

Comment:
Made this deterministic. Snapshot Tests failed on master as well because of this, so felt it was worth solving this, as a next PR will add several more tests, all of which would've been flakey without this 🤷